app/packs/src/apps/admin/UserManagement.js

```diff
@@ -413,17 +420,46 @@ export default class UserManagement extends React.Component {
           this.setState({ messageEditUserModal: result.error });
           return false;
         }
-        this.setState({ showEditUserModal: false, messageEditUserModal: '' });
+        // update this.state.users with result
+        const { users } = this.state;
+        const index = users.findIndex((u) => u.id === user.id);
+        users[index] = result;
+
+        this.setState({
+          users, showEditUserModal: false, messageEditUserModal: '', alertMessage: `User ${user.id} account has been updated.`
+        });
         this.u_email.value = '';
         this.u_firstname.value = '';
         this.u_lastname.value = '';
         this.u_abbr.value = '';
-        this.handleFetchUsers();
         return true;
       });
     return true;
   }
 
+  handleDeleteUser(user) {
+    AdminFetcher.deleteUser({ id: user.id }).then(
+      (result) => {
+        if (result.error) {
+          this.setState({ showEditUserModal: false, alertMessage: result.error });
+          return false;
+        }
+        // remove deleted user from this.state.users
+        const { users } = this.state;
+        const index = users.findIndex((u) => u.id === user.id);
+        if (index > -1) {
+          users.splice(index, 1);
+        }
+        this.setState({
+          showEditUserModal: false,
+          alertMessage: `User with name abbreviation ${user.nameAbbreviation} has been deleted!`,
```

Comment:
user.nameAbbreviation returns undefined (see image below). 
<img width="917" alt="abrev_undefined" src="https://github.com/ComPlat/chemotion_ELN/assets/112618970/3b51ae6e-8067-466c-9c82-56dff8623d06">

The abbreviation is actually stored in **user.initials.**

Reply:
corrected :heavy_check_mark: